src/main/resources/org/jenkinsci/plugins/parallel_test_executor/ParallelTestExecutor/help-testMode.html

```diff
@@ -0,0 +1,21 @@
+<div>
+    Configure how exclusions and inclusions are formed from the JUnit result XML. The test job 
+    receiving the exclusion/inclusion files must be able to handle the format produced by
+    the selected mode.
+    
+    <p>
+    <strong>Parallelize on test classes (Java)</strong><br/>
+    This mode represents the original parallel test executor behavior.
+    Each exclusion/inclusion generates two lines by replacing "." with "/" in the fully qualified test
+    class name and appending ".java" to one line and ".class" to the second line.
+
+    <p>
+    <strong>Parallelize on test case name (Generic)</strong><br/>
+    Each exclusion/inclusion generates one line consisting of the test case name only. This is useful
+    where a tool produces JUnit result XML containing unique test case names without any class prefix. 
+    
+    <p>
+    <strong>Parallelize on class and test case name (Generic)</strong><br/>
+    Each exclusion/inclusion generates one line consisting of the class and test case name on a
+    className.testName format.
```

Comment:
With the drop down descriptor (and a sufficiently new core), you can just give each of these its own `help.html`, which is a bit more modular.

Reply:
Seems to work, but the position of the help handle makes it ugly.	
![Capture d’écran 2024-02-09 à 06 52 27](https://github.com/jenkinsci/parallel-test-executor-plugin/assets/171459/c05c54b0-d82c-498e-b0da-230e5922993b)

Reply:
and this is even worse in the snippet generator, no wonder I didn't see it...
![Capture d’écran 2024-02-09 à 06 55 07](https://github.com/jenkinsci/parallel-test-executor-plugin/assets/171459/377be52c-f9e0-4083-aa75-59a12128a3f4)